src/gate_detect/src/GateDetectorNode.cpp

```diff
@@ -93,7 +125,7 @@ void GateDetectorNode::publishGateImage(GateCoordinates gateCoordinates) {
 
     cv::cvtColor(lineImg, colourMat, CV_GRAY2BGR);
 
-    colourMat = TestUtils::drawGate(colourMat, gateCoordinates);
+    // colourMat = TestUtils::drawGate(colourMat, gateCoordinates);
```

Comment:
Similar comment to above, can we add a rosparam to easily enable/disable this?

Reply:
Fixed